src/scripts/main.js

```diff
@@ -358,3 +358,33 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const tableElement = document.querySelector('.dashboard');
```

Comment:
Ensure that there is an element with the class 'dashboard' in your HTML. If the class name is incorrect or missing, the 'tableElement' will be null, and the code will fail to append rows.